gallery/src/components/NxToggle/NxToggleExample.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState } from 'react';
+
+import { NxToggle } from '@sonatype/react-shared-components';
+
+function NxToggleExample() {
+  // this example uses the `useState` hook for succinctness, but you could also manage the state manually
+  // in a class component
+  const [isWarpOn, setIsWarpOn] = useState(false),
+      [isKrakenOut, setIsKrakenOut] = useState(false),
+      [isShapes, setIsShapes] = useState(false),
+      [isDisabled, setIsDisabled] = useState(false),
+      toggleWarp = () => setIsWarpOn(!isWarpOn),
+      toggleKraken = () => setIsKrakenOut(!isKrakenOut),
+      toggleShapes = () => setIsShapes(!isShapes),
+      toggleDisabled = () => setIsDisabled(!isDisabled);
+
+  return (
+    <>
+      <h3 className="nx-h3">
+        Here's what you can do:
+      </h3>
+      <div>
```

Comment:
No need for this div

Reply:
> No need for this div

https://github.com/sonatype/sonatype-react-shared-components/pull/250/commits/dc33d83ebc6c1322c4c8d41d8e8b9d19003d66e0